client/ayon_core/settings/lib.py

```diff
@@ -203,39 +198,17 @@ def merge_overrides(source_dict, override_dict):
 def get_site_local_overrides(project_name, site_name, local_settings=None):
     """Site overrides from local settings for passet project and site name.
 
+    Warning:
+        This function is not implemented for AYON.
+
     Args:
         project_name (str): For which project are overrides.
         site_name (str): For which site are overrides needed.
         local_settings (dict): Preloaded local settings. They are loaded
             automatically if not passed.
     """
-    # Check if local settings were passed
-    if local_settings is None:
-        local_settings = get_local_settings()
-
-    output = {}
-
-    # Skip if local settings are empty
-    if not local_settings:
-        return output
-
-    local_project_settings = local_settings.get("projects") or {}
-
-    # Prepare overrides for entered project and for default project
-    project_locals = None
-    if project_name:
-        project_locals = local_project_settings.get(project_name)
-    default_project_locals = local_project_settings.get(DEFAULT_PROJECT_KEY)
-
-    # First load and use local settings from default project
-    if default_project_locals and site_name in default_project_locals:
-        output.update(default_project_locals[site_name])
 
-    # Apply project specific local settings if there are any
-    if project_locals and site_name in project_locals:
-        output.update(project_locals[site_name])
-
-    return output
+    return {}
```

Comment:
Should we have some warning here about the local settings not being available yet in AYON?

Reply:
There are TODOs all over the place, the warning would just scare artists about something they can't affect.

Reply:
Ahh, maybe a debug logging for developers then?

Reply:
I really don't think so. For last 3 years it was used on 1 place, and I don't think someone would start to use a function with `Warning:` in docstring and empty implementation.